docs/source/en/_toctree.yml

```diff
@@ -568,5 +568,6 @@
       title: VAE Image Processor
     - local: api/video_processor
       title: Video Processor
-    title: Internal classes
+      title: Internal classes
   title: API
+    
```

Comment:
Why this change?

Reply:
I have referenced the toctree.yml file from the main branch. In previous commits, the CI tests kept failing, so I corrected it.

Reply:
Please run `git restore -s main /path/to/toctree.yaml` and push the changes

Reply:
This should be restored to how it was previously. We can help fix the CI test if it fails

```yaml
    - local: api/video_processor
      title: Video Processor
    title: Internal classes
```